src/cljs/sample_routing/colors.cljs

```diff
@@ -0,0 +1,101 @@
+(ns sample-routing.colors
+  (:require [om.next               :as           om :refer-macros [defui]]
+            [sablono.core          :refer-macros [html]]
+            [sample-routing.filter :refer        [filter-ui]]
+            [taoensso.timbre       :as           log]
+            [compassus.core :as c]))
+
+(defui ColorDetailsItem
+  static om/Ident
+  (ident [_ {:keys [id]}]
+    [:color-details-item/by-id id])
+
+  static om/IQuery
+  (query [_]
+    [:id :description :title])
+
+  Object
+  (render [this]
+    (let [{:keys [id description title]} (om/props this)]
+      (html [:tr {:style {:cursor "pointer"}}
+              [:td id]
+              [:td description]
+              [:td title]]))))
+
+(defui ColorDetails
+  static om/Ident
+  (ident [_ {:keys [color-id]}]
+    [:color-item/by-id color-id])
+
+  static om/IQueryParams
+  (params [this]
+    {
+     :color-desc nil
+     :color-id nil})
+
+  static om/IQuery
+  (query [this]
+    `[({:color/info
+        {:color/header [:color-id :name]
+         :color/details [:id :description :title]}} {:color-id ~'?color-id
+                                                     :color-desc ~'?color-desc})])
+
+  Object
+  (render [this]
+    (let [{{:keys [color/header color/details]} :color/info} (om/props this)]
+      (html [:div 
+             [:h1 (:name header)]
+             [:label "description filter:"
+              (filter-ui)]
```

Comment:
I think what was being proposed in the meeting would mean that we parameterize the filter at this point. I guess we'd need something like...

```
(filter-ui {:filter-attribute :color-desc
              :filter-op :=})
```

Maybe we'd need `:filter-table` too.